app/views/OrderPage/index.tsx

```diff
@@ -0,0 +1,181 @@
+                            valueType="number"
+                        />
+                        <NumberInput
+                            name="quantity"
+                            value={quantity}
+                            onChange={handleQuantityChange}
+                        />
+                        <TextOutput
+                            label="Price (NPR)"
+                            valueType="number"
+                            value={price}
+                        />
+                        <TextOutput
+                            label="Amount (NPR)"
+                            valueType="number"
+                            value={price * quantity}
+                        />
+                    </div>
+                </Container>
+                <div className={styles.orderButton}>
+                    <Button
+                        name={undefined}
+                        onClick={undefined}
+                        variant="secondary"
+                        icons={<BsCashStack />}
+                    >
+                        Cash On Delivery
+                    </Button>
+                </div>
+            </div>
+        </div>
+    );
+}
+
+function OrderPage() {
+    const location = useLocation();
+    const params = new URLSearchParams(location.search);
+    const id = params.get('book');
+    const [quantity, setQuantity] = useState<number>(1);
+    const { data: result, loading } = useQuery<
+        BookByIdQuery,
+        BookByIdQueryVariables
+    >(BOOK_DETAIL, {
+        skip: !id,
+        variables: { id: id ?? '' },
+    });
+
+    const [placeSingleOrder,
+        { data: resp, loading: submitting }] = useMutation<
+            SingleOrderMutation,
+            SingleOrderMutationVariables
+        >(ORDER);
+    const submit = useCallback(() => {
+        const bookId = id ? parseInt(id, 10) : 0;
+        placeSingleOrder({ variables: { bookId, qty: quantity } });
+    }, [id, quantity]);
+
+    const setQuantityChange = (qty: number) => {
+        setQuantity(qty);
+    };
+    const pending = loading || submitting;
+
+    return (
+        <div className={styles.orderList}>
+            {resp && resp.placeSingleOrder && resp.placeSingleOrder.ok && !submitting
+                && (
+                    <>
+                        <Alert
+                            onCloseButtonClick={undefined}
+                            onTimeout={undefined}
+                            name="Alert"
+                            variant="success"
+                        >
+                            Order #
+                            {resp.placeSingleOrder.result?.orderCode}
+                            For Total Price: NPR
+                            {resp.placeSingleOrder.result?.totalPrice}
+                            Order Status:
+                            {resp.placeSingleOrder.result?.status}
+                        </Alert>
+                    </>
+
+                )}
```

Comment:
Remove for now, we'll need to setup alerts separately

Reply:
TODO